packages/react-storage/src/components/FileUploader/hooks/useFileUploader/__tests__/useStorageManager.test.ts

```diff
@@ -0,0 +1,209 @@
+import { act, renderHook } from '@testing-library/react-hooks';
```

Comment:
nit: rename file to useFileUploader.test.ts

Reply:
Just an oversight on my part. Fixed!